appveyor.yml

```diff
@@ -20,6 +20,8 @@ before_deploy:
   - cp OpenRA.Game/OpenRA.ico .
   - appveyor DownloadFile "http://downloads.sourceforge.net/project/nsis/NSIS 2/2.46/nsis-2.46-setup.exe" -FileName nsissetup.exe
   - nsissetup /S /D=%NSIS_ROOT%
+  - appveyor DownloadFile "http://nsis.sourceforge.net/mediawiki/images/archive/1/18/20140806212030!NsProcess.zip" -FileName NsProcess.zip
+  - 7z e NsProcess.zip -o%NSIS_ROOT%

```

Comment:
That should work according to http://help.appveyor.com/discussions/questions/303-how-to-unzip-a-file-in-the-install-script :white_check_mark: We can only test this when tagging a new release.